ogr/ogrsf_frmts/dwg/ogrdwglayer.cpp

```diff
@@ -1179,6 +1179,24 @@ OGRFeature *OGRDWGLayer::TranslateINSERT( OdDbEntityPtr poEntity )
         poFeature->SetField( "BlockAngle", dfAngle );
         poFeature->SetField( "BlockScale", 3, &(oTransformer.dfXScale) );
 
+        OdDbObjectIteratorPtr pEntIter = poRef->attributeIterator();
+        OdDbAttributePtr openAttr;
+
+        CPLJSONObject uAttrData = CPLJSONObject();
+        for (; !pEntIter->done(); pEntIter->step())
+        {
+            openAttr = pEntIter->entity()->objectId().safeOpenObject(OdDb::kForRead);
+            
+            CPLString attrText = TextUnescape( openAttr->textString(), false );
+
+            if ( !openAttr->isInvisible() && openAttr->visibility() != OdDb::kInvisible)
+                uAttrData.Add( CPLSPrintf("%ls", openAttr->tag().c_str()), attrText );
+
+            CPLDebug( "DWG", "TAG: %ls, value %s.", openAttr->tag().c_str(), attrText.c_str() );
```

Comment:
this may have runtime performance impact / spam the log. Remove it or use CPLDebugOnly

Reply:
Thanks for the feedback. Done.

Reply:
Regarding tests, we don't have any dwg example file with blocks. Maybe @mcolmant is able to provide us a very simple dwg file that can be public and that we can use for tests.

Pinging  @mcolmant :-)

Reply:
Of course. Here is a sample file that you can use for your test: https://www.dropbox.com/s/p75jvjv2cu8ri0d/Building_A_Floor_0_Mapwize.dwg?dl=1

Reply:
Thanks @mcolmant. Tests were added.